File: src/top/zedo/telecomcgi/info/GWInfo.java

package top.zedo.telecomcgi.info;

public class GWInfo {
    public String DevType;
    public boolean wifiOnOff;
    public String LANIP;
    public String LANIPv6;
    public String MAC;
    public String ProductCls;
    public String ProductSN;
    public String SWVer;
    public String Vendor;
    public String WANIP;
    public String WANIPv6;
    public String ssid2g;
    public String ssid5g;
    public String wanAcnt;

    @Override
    public String toString() {
        return "GWInfo{" +
                "DevType='" + DevType + '\'' +
                ", wifiOnOff=" + wifiOnOff +
                ", LANIP='" + LANIP + '\'' +
                ", LANIPv6='" + LANIPv6 + '\'' +
                ", MAC='" + MAC + '\'' +
                ", ProductCls='" + ProductCls + '\'' +
                ", ProductSN='" + ProductSN + '\'' +
                ", SWVer='" + SWVer + '\'' +
                ", Vendor='" + Vendor + '\'' +
                ", WANIP='" + WANIP + '\'' +
                ", WANIPv6='" + WANIPv6 + '\'' +
                ", ssid2g='" + ssid2g + '\'' +
                ", ssid5g='" + ssid5g + '\'' +
                ", wanAcnt='" + wanAcnt + '\'' +
                '}';
    }
}

File: src/top/zedo/gatewayapi/info/GWInfo.kt
package top.zedo.gatewayapi.info

data class GWInfo(
    /**
     * 设备类型
     */
    var DevType: String? = null,
    /**
     * wifi开关状态
     */
    var wifiOnOff: Boolean = false,
    var LANIP: String? = null,
    var LANIPv6: String? = null,
    var MAC: String? = null,
    /**
     * 设备型号
     */
    var ProductCls: String? = null,

    /**
     * 序列号
     */
    var ProductSN: String? = null,
    /**
     * 版本号
     */
    var SWVer: String? = null,
    /**
     * 厂商
     */
    var Vendor: String? = null,
    /**
     * 广域网IP
     */
    var WANIP: String? = null,

    /**
     * 广域网IPv6
     */
    var WANIPv6: String? = null,
    var ssid2g: String? = null,
    var ssid5g: String? = null,

    /**
     * 上网账号
     */
    var wanAcnt: String? = null,
)
